ultra/tests/test_evaluate.py

```diff
@@ -471,7 +472,8 @@ def run_experiment(scenario_info, num_agents, log_dir, headless=True):
 
             active_agent_ids = observations.keys() & next_observations.keys()
             loss_outputs = {
-                agent_id: agents[agent_id].step(
+                agent_id: agents[agent_id].act(
```

Comment:
what was the reason for this change?

Reply:
This should not be changed, it should be reverted.

Reply:
Reverted.